content/millennium/r4/financial/charge-item.md

```diff
@@ -69,6 +69,47 @@ All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/r4/St
  `revenue-code`         | None (contains nested extensions) | The type of revenue or cost center providing the product and/or service.
  `unit-price`           | [`Money`]                         | The price of a single unit for the resource.
 
+## Search
+
+<%= beta_tag(action: true) %>
+
+Search for ChargeItems that meet supplied query parameters:
+
+    GET /ChargeItem?:parameters
+
+### Authorization Types
+
+<%= authorization_types(provider: true, patient: false, system: true) %>
```

Comment:
@exopsRamba does provider auth work for chargeItem as of now?

Reply:
I believe so